nagios.go

```diff
@@ -26,3 +49,22 @@ func GetNagiosHistoricalData(r Runner, project, pod string) Task {
 		return r.Run(cmd, path)
 	}
 }
+
+// getResourceNamesBySubstr returns a list of names for the provided resource type that contain
+// the provided string, in the provided project.
+func getResourceNamesBySubstr(project, resource, substr string) ([]string, error) {
```

Comment:
Talked on IRC, staying here for now, may move somewhere else in the future when it will have more callers.